workspaces/vis-network/src/constants.js

```diff
@@ -85,3 +85,24 @@ export const COLORS = Object.freeze({
     }
   }
 });
+
+export const LABELS = Object.freeze({
+  INCOMING: {
+    label: "child of",
+    font: {
+      background: "#EEE"
+    }
+  },
+  OUTGOING: {
+    label: "parent of",
+    font: {
+      background: "#EEE"
+    }
+  },
+  NONE: {
+    label: " ", // A space is used to simulate resetting the edge laebl
```

Comment:
@fraxken I am unable to reset the edge label if i use "" or undefined, only truthy values like " ", 12, "23" are resulting in label updates. any idea why..?